soepy/python/solve/solve_auxiliary.py

```diff
@@ -285,12 +283,17 @@ def construct_emax(
 
         # Calculate flow utility at current period, state, and draw
         flow_utilities = calculate_utilities(
-            attr_dict, educ_level, exp_p, exp_f, optim_paras, corresponding_draws
+            model_params,
```

Comment:
Please let us follow the convention of using `_` for throwaway variables instead, so here `flow_utlities, _, _, _ = calc()`.